homeassistant/components/tuya/sensor.py

```diff
@@ -729,6 +729,22 @@ class TuyaSensorEntityDescription(SensorEntityDescription):
             state_class=SensorStateClass.MEASUREMENT,
         ),
     ),
+    # Curtain
+    # https://developer.tuya.com/en/docs/iot/s?id=K9gf48qy7wkre
+    "cl": (
+        TuyaSensorEntityDescription(
+            key=DPCode.COUNTDOWN_LEFT,
+            name="Remaining Time of Countdown",
+            entity_category=EntityCategory.CONFIG,
+            icon="mdi:timer-outline",
+        ),
```

Comment:
We do not allow for sensors that count down, instead, this needs to be transformed into a timestamp.

Reply:
It will be removed together with the countdown set.